ddtrace/span.py

```diff
@@ -181,6 +184,31 @@ def set_metrics(self, metrics):
     def get_metric(self, key):
         return self.metrics.get(key)
 
+    def set_sampling_priority(self, sampling_priority):
+        """
+        Set the sampling priority.
+
+        0 means that the trace can be dropped, any higher value indicates the
+        importance of the trace to the backend sampler.
+        Default is None, the priority mechanism is disabled.
+        """
+        if sampling_priority is None:
+            self.priority = None
+        else:
+            try:
+                self.priority = int(sampling_priority)
```

Comment:
the only possible values here are 0 or 1?

Reply:
Long term, it can be any `int`.
For now, we have `0` == not sampled, `1` sampled by our sampler. And very soon, `2` explicitly sampled by the user.